tests/schemas/exception.json

```diff
@@ -5,5 +5,5 @@
         "reason": {"type": "string"}
     },
     "additionalProperties": false,
-    "required": ["domain", "reason"]
```

Comment:
I think this is a problem, unless we're going to add a blank reason code also as new exceptions will be added and Android will expect the property.